fvm/src/executor/default.rs

```diff
@@ -38,9 +39,24 @@ impl<K: Kernel> DerefMut for DefaultExecutor<K> {
     }
 }
 
+lazy_static! {
+    static ref EXEC_POOL: yastl::Pool = yastl::Pool::with_config(
+        8,
```

Comment:
This is an arbitrary number; Just for chain-sync we probably don't need more than 1/2, but some users may want this to be higher e.g. to get internal sends in a bunch of historic tipsets.

Reply:
How about using NUM CPUs?

Reply:
Some people have a _lot_ of CPUs. We're already reserving half a gig for stacks here. We'll actually likely want to shrink this.

Reply:
I.e., 128 cores = 8GiB of memory just sitting there.